daphne_worker/src/lib.rs

```diff
@@ -221,13 +224,17 @@ impl DaphneWorkerRouter {
     //
     // TODO Document endpoints that aren't defined in the DAP spec
     pub async fn handle_request(&self, req: Request, env: Env) -> Result<Response> {
+        // Ensure that tracing is initialized.  Some callers may choose to initialize earlier,
+        // but it's safe and cheap to call initialize_tracing() more than once, and this ensures
+        // it's definitely ready for use even if the caller hasn't done anything.
+        initialize_tracing(&env, "DAP_TRACING", "info");
```

Comment:
* What's the name we should use for "info"? The default tracing level, maybe?
* What do you think about having `DaphneWorkerRouter` specify the default logging level?

Reply:
That's not good for the DO case.  I'm thinking maybe both the variable name and the default should just be embedded in initialize_tracing as because of the DOs we are goign to have to call it it like 8 more places.